diff --git a/bot b/bot
index 27d8d92..e350861 100755
--- a/bot
+++ b/bot
@@ -2,7 +2,7 @@
 . ~/.bashrc
 
 NVM_DIR="$HOME/.nvm"
-basePath="$HOME/repos/cdnjs"
+basePath="/cdnjs-sven"
 
 if [ -s "$NVM_DIR/nvm.sh" ] && ! command -v nvm &> /dev/null; then
   . "$NVM_DIR/nvm.sh" --no-use
@@ -22,13 +22,13 @@ if [ "$1" = "debug" ]; then
   export GIT_TRACE=true
 fi
 
-userName="PeterBot"
-userEmail="PeterBot@users.noreply.github.com"
-export GIT_AUTHOR_NAME="${userName}"
-export GIT_AUTHOR_EMAIL="${userEmail}"
-export GIT_COMMITTER_NAME="${userName}"
-export GIT_COMMITTER_EMAIL="${userEmail}"
-export GIT_SSH="$HOME/.ssh/ssh-git.sh"
+# userName="PeterBot"
+# userEmail="PeterBot@users.noreply.github.com"
+# export GIT_AUTHOR_NAME="${userName}"
+# export GIT_AUTHOR_EMAIL="${userEmail}"
+# export GIT_COMMITTER_NAME="${userName}"
+# export GIT_COMMITTER_EMAIL="${userEmail}"
+# export GIT_SSH="$HOME/.ssh/ssh-git.sh"
 
 . "${basePath}/script/commonScript.sh" || exit
 
@@ -88,7 +88,7 @@ while true; do
     run git status
   fi
   print-log "Clean up ..."
-  rm -rf /run/shm/cdnjsCheck/ /run/shm/cdnjs_NPM_temp/
+  rm -rf /cdnjs-sven/cdnjsCheck/ /cdnjs-sven/cdnjs_NPM_temp/
   EndTimestamp="$(date +%s)"
   spent="$((EndTimestamp - StartTimestamp))"
   echo -e "\\nTotal time spent for this build is _${spent}_ second(s)\\n"

