diff --git a/commonScript.sh b/commonScript.sh
index 49328fa..d4cc10b 100644
--- a/commonScript.sh
+++ b/commonScript.sh
@@ -33,8 +33,8 @@ function check() {
   elif [ -z "${oldVer}" ] || [ -z "${newVer}" ]; then
     echo "Version diff not found"
   else
-    rm -rf /run/shm/cdnjsCheck/*
-    tempD="/run/shm/cdnjsCheck/$(date --iso-8601)"
+    rm -rf /cdnjs-sven/cdnjsCheck/*
+    tempD="/cdnjs-sven/cdnjsCheck/$(date --iso-8601)"
     mkdir -p "${tempD}"
     (
       cd "${1}/${oldVer}"
@@ -66,7 +66,7 @@ function check() {
 
 function autoadd() {
   print-log "start to auto add process"
-  cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/"
+  cd "/cdnjs-sven/cdnjs/ajax/libs/"
   for lib in $(git status ./ -uno | command grep -v '\.\.' | command grep package.json | awk '{print $2}' | awk -F'/' '{print $1}'); do
     print-log "Found ${lib}"
     oldVer="$(git diff "${lib}/package.json" | command grep '^\-  "version"' | awk -F'"' '{print $4}')"
@@ -82,14 +82,14 @@ function autoadd() {
         echo "Found ${lib}/${newVer}/.bot_cant_auto_add, pass ..."
         continue
       fi
-      tempD="/run/shm/cdnjsCheck/$(date --iso-8601)"
+      tempD="/cdnjs-sven/cdnjsCheck/$(date --iso-8601)"
       mkdir -p "${tempD}"
       (
-        cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${oldVer}"
+        cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${oldVer}"
         find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${oldVer}_fileList"
       ) &
       (
-        cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${newVer}"
+        cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${newVer}"
         find . -type f -exec chmod -x {} \;
         find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${newVer}_fileList"
       ) &
@@ -108,7 +108,7 @@ function autoadd() {
         rsync -a --delete "${lib}/${newVer}/" "${tempD}/${lib}_${newVer}_x/"
         (
           cd "${tempD}/${lib}_${newVer}_x/"
-          ~/repos/cdnjs/web-minify-helper/minify.sh
+          /cdnjs-sven/web-minify-helper/minify.sh
           find . | sort | command grep -v 'bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${newVer}_x_fileList"
         )
         tmpmd5="$(md5sum "${tempD}/${lib}_${newVer}_x_fileList" | cut -d ' ' -f 1)"
@@ -124,7 +124,7 @@ function autoadd() {
             continue
           fi
           (
-            cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${oldoldVer}"
+            cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${oldoldVer}"
             find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${oldoldVer}_fileList"
           )
           oldoldmd5="$(md5sum "${tempD}/${lib}_${oldoldVer}_fileList" | cut -d ' ' -f 1)"
@@ -154,14 +154,14 @@ function autoadd() {
         echo "Found ${lib}/${newVer}/.bot_cant_auto_add, pass ..."
         continue
       fi
-      tempD="/run/shm/cdnjsCheck/$(date --iso-8601)"
+      tempD="/cdnjs-sven/cdnjsCheck/$(date --iso-8601)"
       mkdir -p "${tempD}"
       (
-        cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${oldVer}"
+        cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${oldVer}"
         find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${oldVer}_fileList"
       ) &
       (
-        cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${newVer}"
+        cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${newVer}"
         find . -type f -exec chmod -x {} \;
         find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${newVer}_fileList"
       ) &
@@ -180,7 +180,7 @@ function autoadd() {
         rsync -a --delete "${lib}/${newVer}/" "${tempD}/${lib}_${newVer}_x/"
         (
           cd "${tempD}/${lib}_${newVer}_x/"
-          ~/repos/cdnjs/web-minify-helper/minify.sh
+          /cdnjs-sven/web-minify-helper/minify.sh
           find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${newVer}_x_fileList"
         )
         tmpmd5="$(md5sum "${tempD}/${lib}_${newVer}_x_fileList" | cut -d ' ' -f 1)"
@@ -195,7 +195,7 @@ function autoadd() {
             continue
           fi
           (
-            cd "${HOME}/repos/cdnjs/cdnjs/ajax/libs/${lib}/${oldoldVer}"
+            cd "/cdnjs-sven/cdnjs/ajax/libs/${lib}/${oldoldVer}"
             find . | sort | command grep -v '.bot_cant_auto_add\|\.map$\|\.ts$\|\.md$' > "${tempD}/${lib}_${oldoldVer}_fileList"
           )
           oldoldmd5="$(md5sum "${tempD}/${lib}_${oldoldVer}_fileList" | cut -d ' ' -f 1)"

